src/stories/nve-button/NveButtonDoc.mdx

```diff
@@ -0,0 +1,88 @@
+import './../css/stories.css';
+
+import {
+  Canvas,
+  Story,
+  Meta,
+  Title,
+  Subtitle,
+  Description,
+  Source,
+  Primary,
+  Controls,
+  Stories,
+} from '@storybook/blocks';
+
+import * as NveButtonStories from './NveButton.stories';
+
+<Meta isTemplate />
+
+<Title />
+<Description />
+<br />
+<br />
+## Varianter
+
```

Comment:
Bruk gjerne teksten i knappen til å beskrive hvilken variant/story du ønsker å vise. Da kan det være at vi slipper å se på koden for å skjønne hva f.eks. variantene heter. Det samme gjelder de andre storyene, som størrelse. Se eksempel fra Shoelace:
![image](https://github.com/NVE/Designsystem/assets/71138449/10cf49ed-c1fa-4a90-aaf7-bdd6e00e217a)